papaparse.js

```diff
@@ -1390,15 +1390,15 @@
 			}
 
 			/**
-             * checks if there are extra spaces after closing quote and newLine without any text
+             * checks if there are extra spaces after closing quote and given index without any text
              * if Yes, returns the number of spaces
             */
-            function extraSpacesLengthAfterQuote(index) {
```

Comment:
Apologies for formatting issue. looks good in multiple IDEs I tried. I will appreciate your help in this one.